rita/src/rita_client/dashboard/neighbors.rs

```diff
@@ -133,6 +150,7 @@ fn generate_neighbors_list(
                 debt: debt_info.debt.clone(),
                 link_cost: exit_route.refmetric,
                 price_to_exit: exit_route.price,
+                stats: *(stats.get(&neigh_route.iface).unwrap()),
```

Comment:
To clean this up you would want to wrap this block in an if let

if let Some(stats_entry) = stats.get(&neigh_route.iface) {
output.push(... // but this time stats_entry is safely handled and not unwrapped) 
}
else {
// nonviable node stuff
}

Reply:
another possibility is to make the if let on line 106 into a match statement then match on both of these things. see the match statement chapter in the rust book.